docs/source-2.0/aws/protocols/aws-json.rst.template

```diff
@@ -195,6 +195,49 @@ In ``awsJson1_1``, servers SHOULD only send the error's
 for both protocols. See `Operation error serialization`_ for full details on how to
 deserialize errors for |quoted shape name|.
 
+.. smithy-trait:: aws.protocols#awsQueryError
+.. _aws.protocols#awsQueryError-trait:
+
+------------------------------------------
+``aws.protocols#awsQueryCompatible`` trait
+------------------------------------------
+
+Summary
+    When using :ref:`awsQuery <aws.protocols#awsQuery-trait>` protocol,
+    custom ``Code`` and ``HTTP response code`` values can be defined for an error response via
+    :ref:`awsQueryError <aws.protocols#awsQueryError-trait>` trait.
+
+    To provide backward compatibility when migrating to :ref:`awsJson1_0 <aws.protocols#awsJson1_0-trait>` protocol,
+    ``awsQueryCompatible`` trait can be used to enable :ref:`awsQueryError <aws.protocols#awsQueryError-trait>` trait
```

Comment:
I think this sentence and the one below should be more specific. I don't see anywhere that we actually document what this trait does... for example:

> This trait adds the ``x-amzn-query-error`` header to error responses so that the service can backward compatibly migrate from awsQuery to awsJson1_0 without removing this metadata from clients. This header is in the form of ``Code;Fault``, where ``Code`` is the value defined in the X, and ``Fault`` is one of ``Sender`` or ``Receiver``.